testsuite/smoke-test/src/fullnode.rs

```diff
@@ -22,22 +22,25 @@ impl Test for LaunchFullnode {
 
 impl NetworkTest for LaunchFullnode {
     fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()> {
+        let runtime = Runtime::new().unwrap();
```

Comment:
I guess the next logical step would be to make these test traits async as well (although that can be done in another step)